docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md

```diff
@@ -4,10 +4,34 @@ displayed_sidebar: "English"
 
 # STOP ROUTINE LOAD
 
+## Description
+
+Stop a Routine Load job.
+
+::: warning
+
+- A stopped Routine Load job can not be resumed.
+- If you only need to pause the Routine Load job, you can execute [PAUSE ROUTINE LOAD](./PAUSE _ROUTINE_LOAD.md).
+
+:::
+
+## Syntax
+
+```SQL
+STOP ROUTINE LOAD FOR [db_name.]<job_name>
+```
+
+## Parameters
+
+| **Parameter** | **Required** | **Description**                                              |
+| ------------- | ------------ | ------------------------------------------------------------ |
+| db_name       |              | The name of the database where the Routine Load job belongs. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/5c4c8d5e-e806-417c-a8ab-518920ec8dc8)
-> to which